mne/io/tests/test_meas_info.py

```diff
@@ -518,7 +518,8 @@ def _test_anonymize_info(base_info):
     new_info = anonymize_info(base_info.copy())
     assert_object_equal(new_info, exp_info)
 
-    new_info = anonymize_info(base_info.copy(), keep_his=True)
+    with pytest.warns(RuntimeWarning, match="keeping 'his_id'"):
+        new_info = anonymize_info(base_info.copy(), keep_his=True)
```

Comment:
so I get a warning all the time even if I actually want to keep HIS?

Reply:
Yes, it used to be `logger.warning` instead of `warn` -- I'll change it to a `logger.info` instead.